dnf-docker-test/features/steps/repo_steps.py

```diff
@@ -65,7 +65,7 @@
 Name:           {{ name }}
 Summary:        {{ summary|default("Empty") }}
 Version:        {{ version|default("1") }}
-Release:        {{ release|default("1") }}%{?dist}
+Release:        {{ release|default("1") }}
```

Comment:
we don't need it at all, I included it there because I used to use it in Fedora. Without this distro-dependent data it will be only better.